From 625d2651258db881d92eb5ffc97cb4f4f1fda239 Mon Sep 17 00:00:00 2001
From: "U. Artie Eoff" <ullysses.a.eoff@intel.com>
Date: Tue, 12 May 2020 11:03:26 -0700
Subject: [PATCH 2/2] test: cast VA_STATSUS_ERROR_UNKNOWN to VAStatus

This fixes a gcc10 compiler error for the
VA_STATUS_ERROR_UNKNOWN 0xFFFFFFFF conversion from
unsigned int to int.

Signed-off-by: U. Artie Eoff <ullysses.a.eoff@intel.com>

Upstream-Status: Backport [https://github.com/intel/intel-vaapi-driver/commit/625d2651258db881d92eb5ffc97cb4f4f1fda239]
Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 test/test.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/test.h b/test/test.h
index 5d3fdc8..fc23ad1 100644
--- a/test/test.h
+++ b/test/test.h
@@ -147,7 +147,7 @@ public:
             status = "VA_STATUS_ERROR_HW_BUSY"; break;
         case VA_STATUS_ERROR_UNSUPPORTED_MEMORY_TYPE:
             status = "VA_STATUS_ERROR_UNSUPPORTED_MEMORY_TYPE"; break;
-        case VA_STATUS_ERROR_UNKNOWN:
+        case VAStatus(VA_STATUS_ERROR_UNKNOWN):
             status = "VA_STATUS_ERROR_UNKNOWN"; break;
         default:
             status = "Unknown VAStatus";
-- 
2.25.4

